src/server/graphql/httpGraphQLHandler.js

```diff
@@ -30,6 +32,13 @@ export default (sharedDataLoader, rateLimiter, sseClients) => async (req, res) =
     res.sendStatus(200)
   }
 
+  if (!isTrebuchetClient) {
+    // act like a regular GraphQL HTTP endpoint
+    const result = await wsGraphQLHandler(connectionContext, req.body)
```

Comment:
nit: it'd be good to pass in `{query, variables}` instead of the whole body so we know we're not using the operationName field that the CLI sends